### ISSUE TITLE
ACUS: change hotel pages to use hotel booking info from configuration settings

### ISSUE THREAD

**Issue by evoskamp** [ggascoigne/amber#163]
Hotel settings added, but not used.

**Comment by ggascoigne** [comment]
You're welcome to migrate from mdx over to tsx like this, but you can also access the config settings by using components that do the appropriate lookup and return the values as elements.  I switched these big blocks of text from tsx over to mdx to make it easier for non-devs to edit, but perhaps that's less of an issue now, or at least less for acus.

**Comment by evoskamp** [comment]
Yeah, I was looking at least effort, as I'm the only one maintaining content and it's pretty much static, except for the hotel links.

**Comment by evoskamp** [comment]
Yeah, since I think the only uses were on this page it didn't feel right to make components for them.
